docs/api.md

```diff
@@ -929,6 +929,7 @@ Shortcut for [page.mainFrame().executionContext().queryObjects(prototypeHandle)]
 
 Triggers a `change` and `input` event once all the provided options have been selected.
 If there's no `<select>` element matching `selector`, the method throws an error.
+If no value in the provided values array matches any `<option>` element, the method throws an error. 
```

Comment:
do i understand correctly, that you now return all the selected options instead of throwing an error? can we add this to the doc?